### PR TITLE
Include DefaultContentType in InteractiveCommand metadata

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@DinoV @tmat @jasonmalinowski

File: src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs [hunk 1]
```diff
@@ -61,8 +61,8 @@ private static string GetDefaultContentTypeName(HostWorkspaceServices workspaceS
 
             if (mefHostServices != null)
             {
-                var registry = mefHostServices.GetExports<IContentTypeRegistryService>().Single().Value;
-                return mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>()
+                var exports = mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>();
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs]: Why was this change needed? You wouldn't want to have unrelated language services impacting the result of this query I think.

Comment (cston) [inline src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs]: 'registry' was not used. Does the call to GetExports<IContentTypeRegistryService>() have side effects?

Comment (cston) [inline src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs]: 'registry' was not used. Does the call to GetExports<IContentTypeRegistryService>() have side effects?

Comment (jasonmalinowski) [inline src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs]: Oh, I see. Too many long type names. My concern here would be if somebody exports their own language service with a content type metadata by accident that weird things might happen. Asking for the exports of the specific language service type would be a bit safer (and cheaper).

Comment (cston) [inline src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs]: Unfortunately mefHostServices.GetExports&lt;IContentTypeLanguageService, ContentTypeLanguageMetadata&gt;() returns 0 exports.
